CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 ## [HEAD](https://github.com/stellargraph/stellargraph/tree/HEAD)
 
-[Full Changelog](https://github.com/stellargraph/stellargraph/compare/HEAD...v0.8.2)
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.9.0...HEAD)
+
+### Major features and improvements
+
+### Breaking changes
+
+### Experimental features
+
```

Comment:
We can potentially pre-fill the paragraph describing experimental features here:

```suggestion

Some new algorithms and features are still under active development, and are available as an experimental preview. However, they may not be easy to use: their documentation or testing may be incomplete, and they may change dramatically from release to release. The experimental status is noted in the documentation and at runtime via prominent warnings.
```